### PR TITLE
majorgeeks.com, domain-status.com, disneybox.com

### PR DESCRIPTION
**Pull request by DandelionSprout**
Example pages:

```
! https://domain-status.com/www/dkfodboldshop.com
domain-status.com##.aunit
! https://www.majorgeeks.com/files/details/snappy_driver_installer.html
majorgeeks.com##.geekad
! http://www.disneybox.com/wiki/%E8%BF%AA%E5%A3%AB%E5%B0%BC%E8%B6%85%E5%87%A1%E5%B0%8F%E4%BB%99%E5%AD%90
disneybox.com##div:has(:scope > script[src*="/pagead/"])
```

### CODE REVIEW

File: dist/placeholder-buster.txt [hunk 1]
```diff
@@ -404,7 +407,9 @@ mythicscribes.com##aside[id^="ai_widget-"]
 itstillworks.com##div.whitecard:has(script[src*="/pagead/"])
 
 # https://github.com/NanoAdblockerLab/NanoContrib/pull/11#issuecomment-463024467
+# https://github.com/NanoAdblockerLab/NanoContrib/pull/38
 ||m.majorgeeks.com/mg/mg.html$subdocument
```
Comment (jspenguin2017) [inline dist/placeholder-buster.txt]: Is this still needed?

**Review by jspenguin2017 — CHANGES_REQUESTED** [review]

File: dist/placeholder-buster.txt (hunk 1 above)
Comment (DandelionSprout) [reply dist/placeholder-buster.txt]: A quick test of that entry's example link, tells me that it's seemingly not needed anymore.

**Review by jspenguin2017 — APPROVED** [review]

**Comment by jspenguin2017** [comment]
`https://domain-status.com/www/dkfodboldshop.com`

**Comment by jspenguin2017** [comment]
`https://www.majorgeeks.com/files/details/snappy_driver_installer.html`

**Comment by jspenguin2017** [comment]
`http://www.disneybox.com/wiki/%E8%BF%AA%E5%A3%AB%E5%B0%BC%E8%B6%85%E5%87%A1%E5%B0%8F%E4%BB%99%E5%AD%90`

**Pull request merged**